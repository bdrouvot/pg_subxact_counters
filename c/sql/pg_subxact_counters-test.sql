CREATE EXTENSION pg_subxact_counters;

begin;
savepoint a;
savepoint b;
savepoint c;
rollback to savepoint a;
commit;

select * from pg_subxact_counters();
