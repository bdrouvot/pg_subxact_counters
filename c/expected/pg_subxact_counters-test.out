CREATE EXTENSION pg_subxact_counters;
begin;
savepoint a;
savepoint b;
savepoint c;
rollback to savepoint a;
commit;
select * from pg_subxact_counters();
 subxact_start | subxact_commit | subxact_abort | subxact_overflow 
---------------+----------------+---------------+------------------
             4 |              1 |             3 |                0
(1 row)

